flink-shaded-curator/flink-shaded-curator-recipes/pom.xml

```diff
@@ -70,6 +70,15 @@ under the License.
 									<include>org.apache.curator:*</include>
 								</includes>
 							</artifactSet>
+							<relocations>
```

Comment:
Do we need this here? I think relocation happens in `flink-runtime`, when it puts curator into its shaded jar.